public/player.js

```diff
@@ -296,7 +318,17 @@ async function refreshAudioSources({ reset } = { reset: false }) {
       audioSources[id].node.buffer = fetchedSources[id];
       audioSources[id].node.loop = true;
       audioSources[id].node.start(startTime + playSegment * segmentDuration);
-      audioSources[id].node.connect(audioContext.destination);
+      audioSources[id].node.connect(audioSources[id].gainNode);
+      audioSources[id].gainNode.connect(audioContext.destination);
+    }
+  });
+
+  // Muting / unmuting
```

Comment:
I note that the same code would work for a volume control button per voice... Anyway, that can be done later on.